README.md

```diff
@@ -502,6 +504,47 @@ export default Base.extend({
 });
 ```
 
+### Deprecation of Authorizers
+
+Authorizers and the session service's `authorize` method are deprecated and
+will be removed from Ember Simple Auth 2.0. The concept seemed like a good idea
+in the early days of Ember Simple Auth, but proved to provide limited value for
+the added complexity. To replace authorizers in an application, simply get the
+session data from the session service and inject it where needed.
+
+In most cases, authorizers are used with Ember Data adapters (refer to the
+[Ember Guides](https://guides.emberjs.com/v3.0.0/models/customizing-adapters/#toc_headers-customization)
+for details on adapters). Replacing authorizers in these scenarios is
+straightforward.
+
+Examples:
+
+```js
+// OAuth 2
+export default DS.JSONAPIAdapter.extend({
+  session: service('session'),
```

Comment:
The `DataAdapterMixin` injects the `session` service already but I guess it makes sense to leave it in here to avoid confusion.